src/test/java/ben/ParserTest.java

```diff
@@ -0,0 +1,27 @@
+package ben;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void invalidCommandTest(){
```

Comment:
Perhaps you can follow featureUnderTest_testScenario_expectedBehavior() for test name?

Reply:
I concur! A description for explaining how it is invalid would be very helpful